### ISSUE TITLE
Pre-defined conformance statement patterns

### ISSUE THREAD

**Issue by Jungyubw** [haffonist/igamt#411]
discuss with Rob about what patterns to be need

**Comment by Jungyubw** [comment]
@robnist 

Rob, This ticket is too old. Do you still need "Pre-defined Patterns". If so, could you clarify the requirements?

Thanks.

**Comment by robnist** [comment]
Yes, we will take the common patterns and make them selectable with one-click, similar to the single coded element binding.

Rob





On: 22 May 2017 08:07, "Jungyubw" <notifications@github.com> wrote:

@robnist<https://github.com/robnist>

Rob, This ticket is too old. Do you still need "Pre-defined Patterns". If so, could you clarify the requirements?

Thanks.

**Comment by robnist** [comment]
Yes, we need pre-defined patterns.
Keep the ticket.
-r
